<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassesDialog</class>
 <widget class="QDialog" name="ClassesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>星期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxWeek">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>请选择授课星期</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>请选择授课星期</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期三</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期四</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期五</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>课时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTime">
       <item>
        <property name="text">
         <string>请选择授课课时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第一节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第二节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第三节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第四节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第五节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第六节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第七节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>第八节</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>授课班级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTeachingclasses">
       <item>
        <property name="text">
         <string>请选择授课班级</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>一班</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>二班</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三班</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>四班</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>五班</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClassesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClassesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
